R/lockbox.R

```diff
@@ -14,48 +14,55 @@
 #' @param file_or_list character or list. A yaml-based lock file or its
 #'    parsed out list format. This set of packages will be loaded into the
 #'    search path and \emph{all other packages will be unloaded}.
+#' @param env character. The name of the entry in the lockfile that contains
+#'    package information.
 lockbox <- function(file_or_list, env = getOption("lockbox.env", "!packages")) {
-  UseMethod("lockbox", file_or_list)
+  UseMethod("lockbox")
 }
 
 #' @export
-lockbox.character <- function(file, env) {
+lockbox.character <- function(file, env = getOption("lockbox.env", "!packages")) {
```

Comment:
You should never need to replicate default parameters in an S3 specialization method.